sqlalchemy_hana/dialect.py

```diff
@@ -475,14 +490,14 @@ class HANAHDBCLIDialect(default.DefaultDialect):
 
     div_is_floordiv = False
     implicit_returning = False
-    postfetch_lastrowid = False
+    postfetch_lastrowid = True
```

Comment:
Has this any kind of implications on tables with a custom sequence?

Reply:
Nope, if you have a sequence, no lasrwoid is used at all. This is verified AFAIK by the SQLAlchemy tests